### ISSUE TITLE
comm_to_RN: consider renaming columns

### ISSUE THREAD

**Issue by Pakillo** [EcologyR/RecruitNetP#19]
Current output: 
A data.frame containing 7 columns with all the information needed for the basic analysis of recruitment networks and canopy-recruit interactions: canopy species (Canopy), recruit species (Recruit), recruitment frequency (fij), Tij = TODO, Pij = TODO, cover of the canopy (cj) and cover of the recruit (ci).

I suggest to be consistent with upper and lowercase column names (e.g. fij, Tij).
And maybe replace i and j by c and r, to represent canopy and recruit? Because otherwise ci or cj are ambiguous, for example

**Comment by jmalcan** [comment]
I agree. I'll check and change where necessary, keeping the uppercase since it allows using lowercase as subindices.
